pcapgo/pcapng.go

```diff
@@ -157,6 +158,17 @@ type NgInterface struct {
 	scaleDown  uint64
 }
 
+// Resolution returns the timestamp resolution of acquired timestamps before scaling to NanosecondTimestampResolution.
+func (i NgInterface) Resolution() (ret gopacket.TimestampResolution) {
+	if i.TimestampResolution.Binary() {
```

Comment:
maybe make TimestampResolution have an Inverse() function that returns a TimestampResolution?